packages/py3_pluggy.rb

```diff
@@ -0,0 +1,22 @@
+require 'package'
+
+class Py3_pluggy < Package
+  description 'Pluggy provides plugin and hook calling mechanisms for Python.'
+  homepage 'https://pluggy.readthedocs.io/'
+  @_ver = '0.13.1'
+  version @_ver
+  license 'MIT'
+  compatibility 'all'
+  source_url 'https://github.com/pytest-dev/pluggy.git'
+  git_hashtag @_ver
+
+  depends_on 'py3_setuptools' => :build
```

Comment:
If you don't include binaries then this needs to be a runtime dependency.

Reply:
That confuses me. Why?

Reply:
Do a fresh crew install and then try to install this package.  It will fail.

Reply:
Doesn't chromebrew install build dependencies when one is building from source?

Reply:
Yes, it will build if you run `crew build py3_pluggy` but not if you run `crew install py3_pluggy`.

Reply:
That's definitely a bug.

Reply:
It's not a bug.  The idea is to not install extra dependencies that are not needed at runtime.

Reply:
But...if binaries are not available, maybe it should bypass the `=> :build` property.

Reply:
Yes, that should happen.